src/main/java/eu/dissco/backend/service/UserService.java

```diff
@@ -47,39 +46,27 @@ private void checkType(JsonApiWrapper request) throws InvalidTypeException {
     }
   }
 
-  public JsonApiData findUser(String id) throws NotFoundException {
-    var optionalUser = repository.find(id);
-    if (optionalUser.isPresent()) {
-      return new JsonApiData(id, TYPE, mapper.valueToTree(optionalUser.get()));
-    } else {
-      throw new NotFoundException();
-    }
+  public JsonApiData findUser(String id) {
```

Comment:
Oeh yes. I think I added the notFoundException because it is part of JSON:API. Could you check if you are still aligned with the specs, I think that was the reason it was working this way